62.md

```diff
@@ -0,0 +1,59 @@
+NIP-62
+======
+
+Request to Vanish
```

Comment:
this mostly called right to be forgotten in the internet. i think we can consider this as a better name. RTBF.

Reply:
We can do either. I was reticent to use "forgotten" because deleting stuff from relays doesn't mean you will be "forgotten". It just means that your data was deleted from that instance or instances that received this event and comply to it.

Reply:
i agree and it's logical. but i think this is same for other stuff on web and internet, and using `forgotten` just makes it easier ti understand for people from everywhere. im more agree with `forgotten` or maybe something between. we can think more about the name.

Reply:
found this: https://en.wikipedia.org/wiki/Right_to_be_forgotten.
they call it vanish as well. we can use one of them and include other one on details of standard. maybe.